Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -56,6 +56,17 @@ public extension Tensor where Scalar: TensorFlowNumeric {
         _Raw.matrixDiag(diagonal: self)
     }
 
+    /// Returns a batched matrix tensor with new batched diagonal values.
+    /// Given the input tensor and diagonal, this operation returns a tensor with the same
+    /// shape and values as the input, except for the specified diagonals of the innermost matrices
+    /// which will be overwritten by the values in diagonal.
+    ///
+    /// Parameter diagonal: A tensor with rank k.
+    @inlinable
+    func setDiagonal(diagonal: Tensor<Scalar>) -> Tensor {
```

Comment:
No `diagonal:` argument label is necessary because the method name already contains "diagonal".
```suggestion
    func setDiagonal(_ diagonal: Tensor<Scalar>) -> Tensor {
```